src/public/inventory.ts

```diff
@@ -133,5 +133,32 @@ export const inventoryPublicHandlers: PublicNatsHandler[] = [
         {
             queue: SERVICE_NAME
         }
+    ],
+    [
+        "PUT",
+        "inventory.*",
+        async (subscription: Subscription): Promise<void> => {
+            for await (const message of subscription) {
+                try {
+                    const natsConnection = getConnection();
+                    const { body } = jsonCodec.decode(
+                        message.data
+                    ) as AirlockPayload;
+
+                    const response = await natsConnection.request(
+                        "item-store.transfer_inventory_item",
+                        jsonCodec.encode(body)
+                    );
+
+                    message.respond(response.data);
+                } catch (err) {
+                    message.respond(
+                        jsonCodec.encode({
+                            error: err.message
+                        })
+                    );
+                }
+            }
+        }
```

Comment:
we need to set a queue group for this handler, otherwise all similar handlers will be called if we have multiple instnaces of this handler running. Look at the other handlers above to see how to configure the queue group.

The rule is: if a handler does a read: don't use a queue group, if it does a write, use a queue group